Dshop/apps/products_catalogue/tests/test_categories.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from apps.products_catalogue.models import Category
+from django.urls import reverse
+
+
+NON_EXISTING_SLUG = "xzcvbnmzxc"
+
+
+@pytest.mark.django_db
+def test_get_categories_list(client):
+    cat_1 = Category.objects.create(name="Koszule")
+    cat_2 = Category.objects.create(name="Buty")
+    cat_3 = Category.objects.create(name="Druty")
+    url = reverse("category-list")
+    response = client.get(url)
+    categories = response.context["categories"]
+    assert response.status_code == 200
+    assert (
+        cat_1 in categories and
+        cat_2 in categories and
+        cat_3 in categories
+    )
+
+
+@pytest.mark.django_db
+def test_get_empty_categories_list(client):
+    url = reverse("category-list")
+    response = client.get(url)
+    assert response.status_code == 200
+    categories = response.context["categories"]
+    assert len(categories) == 0
+
+
+@pytest.mark.django_db
+def test_get_non_active_categories_list(client):
```

Comment:
would be nice to add similar test where 1 is active and 1 isn't

Reply:
Or simply modify this one to have one active.